source/renderer/app/utils/formatters.js

```diff
@@ -19,7 +13,30 @@ export const formattedWalletAmount = (
   let formattedAmount = '';
   if (long) {
     formattedAmount = amount.toFormat(DECIMAL_PLACES_IN_ADA);
-  } else if (amount.isZero()) {
+  } else {
+    formattedAmount = shortNumber(amount);
+  }
+  const { decimalSeparator } = BigNumber.config().FORMAT;
+  if (!long && decimalSeparator !== '.') {
+    // Only BigNumber.toFormat() method is applying correct separators.
+    // Since this method is not used for condensed format (long = false)
+    // the correct number format has to be applied manually.
+    formattedAmount = formattedAmount.split('.').join(decimalSeparator);
+  }
+  if (withCurrency) formattedAmount += ' ADA';
```

Comment:
What about using ES6 syntax?
if (withCurrency) formattedAmount = `${formattedAmount} ADA`;

Reply:
@yakovkaravelov can you please fix this?